scripts/table_costs_efficiencies.py

```diff
@@ -10,7 +10,7 @@
     ``results/_tables/technical_and_cost_assumptions_{scenario_key}.csv``: target path for
     the table.
 logfile : str
-    ``logs/{scenario}.log``: path to logfile
+    ``results/_tables/technical_and_cost_assumptions_{scenario_key}.log``: path to logfile
```

Comment:
I agree that using {scenario} everywhere would be more consistent. I had used {scenario_key} in the docstring because that was the output name in the Snakefile. I've now changed both (see [a833d84](https://github.com/rl-institut/oemof-B3/pull/279/commits/a833d849c5be23f578057f8a887a96e27d78c35d)). Perhaps we could also change that in the cost-efficiencies-table in the input data?

Reply:
Thank you for your changes. Tested it and it works.

Regarding `scenario_key` in the raw data, you have picked up a good point. I took a look and saw that we don't have any guidance yet regarding scenario setup in the docs. I think for using `oemof-B3` it would be helpful to describe in a chapter "scenario requirements" or "scenario setup" how the scenario `yml` file is structured and how it is linked to the raw data. Questions that arise: What distinction in the scenario keys is minimally necessary for oemof-B3 to run? Which `scenario_key` belongs to which raw data file? So far there is only one sentence (deprecated though) at the bottom of [this paragraph](https://oemof-b3.readthedocs.io/en/latest/model_structure.html#oemof-b3-resources). I would open up a new issue with this, what do you think @monika-o ?

Reply:
That's a very good idea, I totally agree with you! It would be great if we had a chapter about the scenario setup.

I also agree that we should move that to a different pull request. Could you merge this one now @MaGering ?